### PR TITLE
Docs for new permissions of 11.2

### PR DESCRIPTION
**Pull request by bluedeepimpact**
some corrections and improvements in the documentation for the newly introduced global permissions

### CODE REVIEW

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 1]
```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
 
 ![global-roles-in-openproject](image-20210308171607279.png)
 
-You can choose between these global roles:
+You can choose between these global permissions:
 
-- **Create project**: Assign this role to users to enable them to create new projects without being system administrator. 
-- **Create and edit users**: Assign this role to users who should be able to create or invite new users and edit their profiles in a limited way. 
-  Users with this role can see all users of your OpenProject instance and can add users and edit the name, username, email address and language of a user. They can't delete or lock users. They can only see and add users to projects where they have permissions to see project members.
+- **Create project**: Assign this permission to users to enable them to create new projects without being system administrator.
```
Comment (wielinde) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]: That sentence is irritating a bit as we do not assign permissions to users. We assign permissions to roles and roles to project memberships.

Simple suggestion:

**Create project**: With this permission users can create new projects even when they are not system administrators.

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 2]
```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
...
 ![global-roles-in-openproject](image-20210308171607279.png)
 
-You can choose between these global roles:
+You can choose between these global permissions:
 
-- **Create project**: Assign this role to users to enable them to create new projects without being system administrator. 
-- **Create and edit users**: Assign this role to users who should be able to create or invite new users and edit their profiles in a limited way. 
-  Users with this role can see all users of your OpenProject instance and can add users and edit the name, username, email address and language of a user. They can't delete or lock users. They can only see and add users to projects where they have permissions to see project members.
+- **Create project**: Assign this permission to users to enable them to create new projects without being system administrator. 
+  You can change the [project role](../../../user-guide/members) a user with this permission is assigned in a newly created project [this way](../../system-settings/project-system-settings/#settings-for-new-projects).
```
Comment (wielinde) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]: I don't understand that sentence. Ah, you probably want to say something about the default role of project creators.

Suggestion:

You can set a default role that they will have in the created project [here](../../system-settings/project-system-settings/#settings-for-new-projects).

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 3]
```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
...
+- **Create and edit users**: Assign this permission to users who should be able to create or invite new users and edit their profiles in a limited way.
```
Comment (wielinde) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]: I'd put the "limited way" in an extra sentence.

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 4]
```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
...
-- **Create and edit users**: Assign this role to users who should be able to create or invite new users and edit their profiles in a limited way. 
-  Users with this role can see all users of your OpenProject instance and can add users and edit the name, username, email address and language of a user. They can't delete or lock users. They can only see and add users to projects where they have permissions to see project members.
+- **Create project**: Assign this permission to users to enable them to create new projects without being system administrator. 
+  You can change the [project role](../../../user-guide/members) a user with this permission is assigned in a newly created project [this way](../../system-settings/project-system-settings/#settings-for-new-projects).
+- **Create and edit users**: Assign this permission to users who should be able to create or invite new users and edit their profiles in a limited way. 
+  Users with this permission can see all users of your OpenProject instance and can add users and edit the name, username, email address and language of a user. They can't delete or lock users.
```
Comment (wielinde) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]: That sentence is a bit long and needs to be cut into multiple sentences, I believe. Also the focus on user visibility as the first bullet is irritating. It is a secondary, though important, side effect.

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 5]
```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
...
+  They can only see the project membership of users for projects in which they have permission to see the members (e.g. as Project admin or Member). They can only manage project membership of users for projects in which they have permission to manage members (e.g. as Project admin).
   The user profile will look like this for them (user name and email address were redacted): ![create-and-edit-users-role](image-20210308180635158.png)
-- **Create, edit and delete placeholder users**: Assign this role to users (e.g. project admins) who should be able to manage [placeholder users](../placeholder-users). 
-  Users with this role can see all placeholder users in your OpenProject instance and can create, edit and delete placeholder users. They can only see and add placeholder users to projects where they have permissions to see project members.
+- **Create, edit and delete placeholder users**: Assign this permission to users (e.g. project admins) who should be able to manage [placeholder users](../placeholder-users).
```
Comment (wielinde) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]: I believe in English there comes a comma before the "and" as it is a enumeration. So "Create, edit, and delete placeholder users". I will fixed that in the app code just now. Please update this in the docs, too.

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 6]
```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
...
+  They can only see the project membership of users for projects in which they have permission to see the members (e.g. as Project admin or Member). They can only manage project membership of users for projects in which they have permission to manage members (e.g. as Project admin).
   The user profile will look like this for them (user name and email address were redacted): ![create-and-edit-users-role](image-20210308180635158.png)
-- **Create, edit and delete placeholder users**: Assign this role to users (e.g. project admins) who should be able to manage [placeholder users](../placeholder-users). 
-  Users with this role can see all placeholder users in your OpenProject instance and can create, edit and delete placeholder users. They can only see and add placeholder users to projects where they have permissions to see project members.
+- **Create, edit and delete placeholder users**: Assign this permission to users (e.g. project admins) who should be able to manage [placeholder users](../placeholder-users). 
+  Users with this permission can see all placeholder users in your OpenProject instance and can create, edit and delete placeholder users.
```
Comment (wielinde) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]: Again, needs a comma bofore the "and". 

Again, I'd change the order. Seeing other placeholder users is not the main focus of that permission.

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md [hunk 7]
```diff
@@ -77,13 +77,16 @@ To create a global role tick the box "Global Role" when [creating a new role](#c
...
+  They can only see the project membership of placeholder users for projects in which they have permission to see the members (e.g. as Project admin or Member). They can only manage project membership of placeholder users for projects in which they have permission to manage members (e.g. as Project admin).
```
Comment (wielinde) [inline docs/system-admin-guide/users-permissions/roles-permissions/README.md]: Maybe that sentence needs some "Hint:" or "Watch out:" as that sentence is meant to explain why placeholder users are not visible/editable **although** you have the right to create, edit, and delete them.

**Review by wielinde — CHANGES_REQUESTED** [review]
Hey @ML-OpenP, I went through your doc changes. You caught quite some stuff. Nice! 

Please see me comments as suggestions for improvements of text readability and focus.

File: docs/system-admin-guide/users-permissions/roles-permissions/README.md (hunk 5 above)
Comment (bluedeepimpact) [reply docs/system-admin-guide/users-permissions/roles-permissions/README.md]: @wielinde deepl.com disagrees :-P
But if you're sure I'm fine with that

Comment (wielinde) [reply docs/system-admin-guide/users-permissions/roles-permissions/README.md]: I did some googling. apparently both ways are correct

Comment (bluedeepimpact) [reply docs/system-admin-guide/users-permissions/roles-permissions/README.md]: Thank you, Wieland! Yeah, me too. Sorry for bothering you with this!

**Pull request merged**